digitalpublicgoods/commcare.json

```diff
@@ -0,0 +1,97 @@
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "GDPR",
+      "HIPPA"
+    ],
+    "adherenceSteps": [
+      "1. Terms of Service: https://www.dimagi.com/terms/latest/tos/",
+      "2. Privacy Policy: https://www.dimagi.com/terms/latest/privacy/",
+      "3. Business Agreement: https://www.dimagi.com/terms/latest/ba/",
+      "4. Acceptable Use Policy: https://www.dimagi.com/terms/latest/aup/"
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "The CommCare application is built on OpenRosa standard tools and protocols, and leverages many different open-source technologies. ",
+      "Protocols are represented using XForms, a W3C backed standard approach to more advanced forms that support nested and repeatable elements, decision support, advanced validation, and extensive user interface controls."
+    ],
+    "evidenceStandardSupport": [
+      ""
+    ],
+    "implementBestPractices": "Yes",
+    "bestPracticesList": [
+      "Our design and style guide has the goal to ensure that our experiences meet the standards for accessibility required by WCAG AA, Section 508 and European Standards. Link: https://www.commcarehq.org/styleguide/atoms/"
+    ]
+  },
+  "doNoHarm": {
+    "preventHarm": {
+      "stepsToPreventHarm": "Yes",
+      "additionalInfoMechanismProcessesPolicies": "https://www.dimagi.com/blog/mobile-data-collection-nhs-guidelines/, \nhttps://www.dimagi.com/terms/latest/privacy/"
+    },
+    "dataPrivacySecurity": {
+      "collectsPII": "Yes",
+      "typesOfDataCollected": [
+        "CommCare allows for flexible data collection and thus partners can dictate the data they store. In the context of routine immunization demographic and health data will be collected and stored"
+      ],
+      "thirdPartyDataSharing": "No",
+      "dataSharingCircumstances": [
+        ""
+      ],
+      "ensurePrivacySecurity": "Yes",
+      "privacySecurityDescription": "https://confluence.dimagi.com/display/commcarepublic/CommCare+Technical+Overview?preview=%2F12223445%2F56885462%2FData+Security+Overview.pdf"
+    },
+    "inappropriateIllegalContent": {
+      "collectStoreDistribute": "No",
+      "type": "",
+      "contentFilter": "",
+      "policyGuidelinesDocumentationLink": "",
+      "illegalContentDetection": "",
+      "illegalContentDetectionMechanism": ""
+    },
+    "protectionFromHarassment": {
+      "userInteraction": "Yes",
+      "addressSafetySecurityUnderageUsers": "Yes",
+      "stepsAddressRiskPreventSafetyUnderageUsers": [
+        "CommCare doesn't have any technical measures to address the safety and security of underage users. We require the organizations who use CommCare to comply with applicable laws about minimum age requirements of end users."
+      ],
+      "griefAbuseHarassmentProtection": "No",
+      "harassmentProtectionSteps": [
+        ""
+      ]
+    }
+  },
+  "locations": {
+    "developmentCountries": [
```

Comment:
Though not required, it would be really nice to have countries listed here. Thoughts? @nathanbaleeta